scripts/generate_dump

```diff
@@ -285,6 +285,11 @@ main() {
     save_redis "2" "COUNTERS_DB"
 
     save_cmd "docker exec -it syncd saidump" "saidump"
+    
+    local sai_dump_filename="/tmp/syncd_sai_dump"
```

Comment:
should be only on mellanox platform since it is the only platform that supports this.